static/src/javascripts/projects/journalism/views/campaignForm.html

```diff
@@ -0,0 +1,102 @@
+<details class="campaign campaign--snippet">
+
+    <summary class="campaign--snippet__header">
+        <div class="kicker">
+            <div class="campaign--snippet__heading-logo">
+                <div class="speech-bubble">
```

Comment:
Minor: can we use sth more semantic, like `campaign__kicker`?

Reply:
Instead of heading-logo? `<div class="campaign--snippet__kicker">`